howso/engine/trainee.py

```diff
@@ -717,25 +678,21 @@ def metrics(self) -> "Metrics":
         else:
             raise ValueError("Client must have 'get_trainee_metrics' method")
 
-    def set_random_seed(self, seed: Union[int, float, str]) -> None:
+    def set_random_seed(self, seed: int | float | str):
         """
         Set the random seed for the trainee.
 
         Parameters
         ----------
-        seed : int or float or str
+        seed : int | float | str
```

Comment:
Are the vertical bars in the Numpy-style doc strings? I thought we preferred the more natural english "or". Serious question. I don't recall.